android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/enums/AppCapabilityTypeTests.java

```diff
@@ -0,0 +1,63 @@
+package com.smartdevicelink.test.rpc.enums;
+
+import com.smartdevicelink.proxy.rpc.enums.AppCapabilityType;
+
+import junit.framework.TestCase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class AppCapabilityTypeTests extends TestCase {
+
+    /**
+     * Verifies that the enum values are not null upon valid assignment.
+     */
+    public void testValidEnums () {
+        String example = "VIDEO_STREAMING";
+        AppCapabilityType enumVideoStreaming = AppCapabilityType.valueForString(example);
+
+        assertNotNull("VIDEO_STREAMING returned null", enumVideoStreaming);
+    }
+
+    /**
+     * Verifies that an invalid assignment is null.
+     */
+    public void testInvalidEnum () {
+        String example = "VidEOs_Treamin";
+        try {
+            AppCapabilityType temp = AppCapabilityType.valueForString(example);
+            assertNull("Result of valueForString should be null.", temp);
+        }
+        catch (IllegalArgumentException exception) {
+            fail("Invalid enum throws IllegalArgumentException.");
+        }
+    }
+
+    /**
+     * Verifies that a null assignment is invalid.
+     */
+    public void testNullEnum () {
+        String example = null;
+        try {
+            AppCapabilityType temp = AppCapabilityType.valueForString(example);
+            assertNull("Result of valueForString should be null.", temp);
+        }
+        catch (NullPointerException exception) {
+            fail("Null string throws NullPointerException.");
+        }
+    }
+
+    /**
+     * Verifies the possible enum values of AmbientLightStatus.
```

Comment:
@kostyaBoss In this comment on line 52, you can change this from AmbientLightStatus to AppCapabilityType.